imports/ui/helpers.coffee

```diff
@@ -11,8 +11,8 @@ UI.registerHelper 'formatDateRange', (dateRange)->
   return formatDateRange(dateRange)
 
 UI.registerHelper 'incidentToText', (incident) ->
-  if @cases
-    incidentDescription = pluralize("case", @cases)
+  if incident.cases
```

Comment:
Oh that must have been incidental.
I also didn't realize we already had a helper registered. I'll combine `incidentTypeWithCountAndDisease` with this.